### PR TITLE
PAPP-29962 Sqlite update

### PR DESCRIPTION
**Pull request by ighelani-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,60 +64,60 @@ Run a query against a table or tables in the database
...
-action\_result\.data\.\* | string | 
-action\_result\.summary\.total\_rows | numeric | 
-action\_result\.message | string | 
-summary\.total\_objects | numeric | 
-summary\.total\_objects\_successful | numeric |   
+DATA PATH | TYPE | CONTAINS | EXAMPLE VALUES
+--------- | ---- | -------- | --------------
+action_result.status | string |  |   success 
+action_result.parameter.vault_id | string |  `vault id`  |   da39a3ee5e6b4b0d3255bfef95601890afd80709 
+action_result.parameter.format_vars | string |  |   a
```
Comment (splunk-jessica) [inline README.md:82]: looks like there are some typos in the readme where the `|` comes before the table value

**Review by splunk-jessica — APPROVED** [review]

**Review by splunk-jessica — COMMENTED** [review]
why is there no pipeline run for this? Pretty sure this is Splunk-supported app so we should have integration tests for this right? Also, Splunk-supported apps should have "Splunk" as the publisher in the app json, not "Phantom." In addition, please update the copyright date to 2023.

**Review by tonyc-phantom — APPROVED** [review]

File: sqlite.json [hunk 2]
```diff
@@ -8,13 +8,13 @@
     "logo_dark": "logo_sqlite_dark.svg",
     "product_name": "SQLite",
     "product_version_regex": ".*",
-    "publisher": "Phantom",
-    "license": "Copyright (c) 2017-2022 Splunk Inc.",
-    "app_version": "2.0.6",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2017-2023 Splunk Inc.",
+    "app_version": "2.1.6",
```
Comment (ishans-crest) [inline sqlite.json:13]: should be 2.1.0

**Review by ishans-crest — APPROVED** [review]

File: sqlite.json (hunk 2 above)
Comment (ighelani-crest) [reply sqlite.json]: changed the version

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**